src/main/groovy/wooga/gradle/snyk/tasks/SnykTask.groovy

```diff
@@ -91,7 +91,8 @@ abstract class SnykTask extends DefaultTask
                 }
             }
         })
-        handleExitCode(execResult.exitValue)
+
+        handleExitCode(0)
```

Comment:
Hmm; I think we should instead handle this within the function, by first printing the corresponding error message, and just not asserting if its not 0.

Reply:
That must be a merge error.